### ISSUE TITLE
Oscar 2.0

### ISSUE THREAD

**Issue by jayvdb** [sayonetech/oscar_partner#1]
I believe most of the features of this extension are now in Oscar 2.0.  Is there anything missing?

A note on the README would be useful.

**Comment by renjithraj2005** [comment]
@jayvdb yes. This was developed long time back and this is not maintained now

**Comment by jayvdb** [comment]
Do you know if all features have been implemented in Oscar 2?

If so, archive repo, or add a note on README?
